app/presenters/tree_builder_tags.rb

```diff
@@ -45,16 +49,18 @@ def root_options
 
   def x_get_tree_roots(count_only, _options)
     # open node if at least one of his kids is selected
-    @categories.each do |c|
-      open_node("cl-#{to_cid(c.id)}") if contain_selected_kid(c)
+    if !count_only && (@edit.present? || @filters.present?)
+      @categories.each do |c|
+        open_node("cl-#{to_cid(c.id)}") if contain_selected_kid(c)
+      end
     end
     count_only_or_objects(count_only, @categories)
   end
 
   def x_get_classification_kids(parent, count_only)
+    return parent.entries.size if count_only
     kids = parent.entries.map do |kid|
-      kid_id = "#{parent.name}-#{kid.name}"
-      select = (@edit && @edit.fetch_path(:new, :filters, kid_id)) || (@filters && @filters.key?(kid_id))
+      select = selected_entry_value(parent, kid)
```

Comment:
I should have asked this before... was this supposed to be a boolean or the value for that key?  Seems like it should have been the value, but not sure.